frontend/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# testing
+/coverage
```

Comment:
really needed for the front? it's already in the main `.gitignore`

Reply:
The gitignore is the create-react-app default. If we see this dir as a kind of sub-project I think it makes sense to leave this `.gitignore` here for frontend-only files.

Plust the rules in the `.gitignore` in master do not apply here, instead of `/node_modules` it should be `frontend/node_modules`, right?

Reply:
I'm not sure if the folder is like a "subproject"...

I use to like small `.gitignore` files... with as fewer rules as possible, so I like the idea of avoiding duplicities; anyhow this is more like an opinion, so if you do not share it, feel free to disregard it :dagger: 

About `node_modules` rule:
if it should be in the main `.gitignore` it should be `frontend/node_modules` but since it makes sense to be in the `frontend/.gitignore` it needs to relative to the `frontend` dir.